### DISCUSSION TITLE
add whitespace between 漢字 and よみ in Dictionary

### DISCUSSION THREAD

**Discussion by veryoldone** [fabd/kanji-koohii#269] [1. Feedback & Feature Requests]
Hello Fabrice,

Many, many thanks for this superb service. I've found Kanji Koohii truly inspirational.

This is a minor suggestion, probably only useful to me, but anyway:

My workflow for adding cards I've studied includes copying a few dictionary entries, reformatting them <漢字、よみ　English>  to suit my preference, and pasting into the Story field under my story and the onyomi and kunyomi.

There used to be a whitespace between the kanji and the yomi in Dictionary. That whitespace enabled me to use a macro to format all the copied entries — a big timesaver for me.  

About a year ago that whitespace disappeared. Any chance you'll add it back?

Thank you.

**Comment by fabd** [comment]
Hi veryoldone 

I just did a small update on the live site. To not lose track of the change I put a `&nbsp;` html entity in the html template (as opposed to just adding a space which can often be removed by "pretty code" extensions) - but I think the browser renders it as an actual space, not 100% sure (there are also those annoying non-breakable-space  special unicode characters which are usually unwanted, and I'm not 100% sure if the actual character code ends up one of those -  I did a simple search & replace for space in gedit text editor and it matched the space after the kanji ... but maybe gedit also accounts for different type of "space like" characters).

Aaaaanyways... if this doesn't work we can easily tweak it to something else :)

Let me know of the issue is solved so I can merge the code (although since it is a small change with no side effects, the live site IS already updated with the change).

You did well to bring it up again, even though it is a small thing, it is very easy to lose track of those things, as the list goes on and on, and I do this in my spare time.

And thank you for the support :)
